#166272327 Deploy app to Heroku

EMacco opened the pull request:
#### What does this PR do?
Finish up app deployment to Heroku

#### Description of Task to be completed?
- Add sequelize-cli as a dependency
- Add script to run migration after the app has been built
- Set env variables in app.json
- Set addons in app.json

#### How should this be manually tested?
N/A

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/166272327

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A

EMacco committed:
chore-[166272327] Deploy base app to heroku

- Add sequelize-cli as a dependency
- Add script to run migration after the app has been built
- Set env variables in app.json
- Set addons in app.json

[Finishes #166272327]

EMacco committed:
chore-[166272327] Deploy base app to heroku

- Add sequelize-cli as a dependency
- Add script to run migration after the app has been built
- Set env variables in app.json
- Set addons in app.json

[Finishes #166272327]

EMacco committed:
Merge branch '166272327-ch-deploy-app-to-heroku' of https://github.com/andela/kingsmen-ah-backend into 166272327-ch-deploy-app-to-heroku

adex001 reviewed (commented):
Looks good to me. You need to resolve merge conflict.

fidelisojeah commented on app.json:
```diff
@@ -2,11 +2,20 @@
   "name": "kingsmen-ah-backend",
   "description": "A community of like minded authors that fosters inspiration and innovation by leveraging the modern web.",
   "scripts": {},
```
include the following script 
    `"postdeploy": "heroku config:set HOST_NAME=${HEROKU_APP_NAME}.herokuapp.com --app ${HEROKU_APP_NAME} && npm run refreshdb"`
 we would look more into it later

EMacco replied:
Ok Thanks

fidelisojeah commented on app.json:
```diff
@@ -2,11 +2,20 @@
   "scripts": {},
-  "env": {},
+  "env": {
+    "SALT": {
```
include the following env
```json
"HEROKU_APP_NAME": {
      "required": true
    },
    "HEROKU_PARENT_APP_NAME": {
      "required": true
    }
```

EMacco replied:
Will do. I am guessing I will have to set those in Heroku environment too?

EMacco replied:
Ok I understand now. Thanks

EMacco committed:
chore-[166272327] Deploy base app to heroku

    - Add sequelize-cli as a dependency
    - Add script to run migration after the app has been built
    - Set env variables in app.json
    - Set addons in app.json

    [Finishes #166272327]

EMacco committed:
chore-[166272327] Deploy base app to heroku

    - Add sequelize-cli as a dependency
    - Add script to run migration after the app has been built
    - Set env variables in app.json
    - Set addons in app.json

    [Finishes #166272327]

EMacco committed:
Merge branch '166272327-ch-deploy-app-to-heroku' of https://github.com/andela/kingsmen-ah-backend into 166272327-ch-deploy-app-to-heroku

EMacco committed:
Merge branch 'develop' into 166272327-ch-deploy-app-to-heroku

despeauxz reviewed (approved):
Everything looks good. kudos man
